pom.xml

```diff
@@ -622,7 +622,7 @@
         <jackson.databind.version>2.13.4.2</jackson.databind.version>
         <kraal.version>0.0.16</kraal.version>
         <kotlin.stdlib.version>1.3.31</kotlin.stdlib.version>
-        <prometheus.jmx.agent.version>0.14.0</prometheus.jmx.agent.version>
+        <prometheus.jmx.agent.version>0.17.2</prometheus.jmx.agent.version>
```

Comment:
@IsuruMaduranga Changing this place will not enough. Change this in gateway and gateway.bat files also

Reply:
fixed